src/main/java/pl/jakubdrozdz/phishingvalidator/sms/utils/SMSUtils.java

```diff
@@ -0,0 +1,53 @@
+package pl.jakubdrozdz.phishingvalidator.sms.utils;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import pl.jakubdrozdz.phishingvalidator.sms.model.SMSRegistrationRequest;
+import pl.jakubdrozdz.phishingvalidator.sms.service.SMSService;
+import pl.jakubdrozdz.phishingvalidator.subscriber.exception.SubscriberNotExistingException;
+import pl.jakubdrozdz.phishingvalidator.subscriber.service.SubscriberService;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Slf4j
+@RequiredArgsConstructor
+public class SMSUtils {
+    private final SMSService smsService;
```

Comment:
Czy ta zmienna jest do czegoś używana?

Reply:
Zostało to dodane w tym commit'cie 0e0de14e53c1c9784cf0bbb30263302f1f3f063d i zapomniałem usunąć. Tak jak wspomniałem Kasi Mathias, wyjeżdżałem na urlop i kończyłem to zadanie już w podróży i zapomniałem zrobić cleanup